.github/workflows/tests.yaml

```
@@ -27,7 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kind-k8s-version: [1.26.15, 1.27.13, 1.28.9, 1.29.4, 1.30.0]
+        kind-k8s-version: [1.27.13, 1.28.9, 1.29.10, 1.30.6, 1.31.2]
```

It looks like there are some slightly newer versions available now for [1.28](https://hub.docker.com/r/kindest/node/tags?name=1.28) and [1.27](https://hub.docker.com/r/kindest/node/tags?name=1.27):

```suggestion
        kind-k8s-version: [1.27.16, 1.28.15, 1.29.10, 1.30.6, 1.31.2]
```